src/Components/Goods/DeliveryInfo.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './Goods.scss';
+export default class DeliveryInfo extends React.Component {
```

Comment:
```suggestion
import './Goods.scss';

export default class DeliveryInfo extends React.Component {
```
구분되는 요소들끼리는 한줄 띄워주세요

Reply:
네 알겠습니다